nbextensions/bulkImportCell/jobsData.js

```diff
@@ -0,0 +1,293 @@
+define(['common/format'], (format) => {
```

Comment:
This populates the job status table with a set of jobs, one in each state. The same data is used for tests and when running the bulk import cell on CI.